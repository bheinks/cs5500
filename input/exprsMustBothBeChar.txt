(* 
  Expressions must both be int, or both char, or both boolean
*)

program p;

var i : integer;
    b : boolean;
    c : char;

begin
  b := (i > 5) or (c = 65);
  write('!')
end.

