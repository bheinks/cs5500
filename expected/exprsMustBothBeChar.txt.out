Line 12: Expressions must both be int, or both char, or both boolean
